---a/net/minecraft/entity/ai/EntityAIWander.java
+++b/net/minecraft/entity/ai/EntityAIWander.java
@@ -3,8 +3,8 @@
 import net.minecraft.entity.EntityCreature;
 import net.minecraft.util.Vec3;
 
-public class EntityAIWander extends EntityAIBase
-{
+public class EntityAIWander extends EntityAIBase {
+
     private EntityCreature entity;
     private double xPosition;
     private double yPosition;
@@ -14,41 +14,33 @@
     private boolean field_179482_g;
     private static final String __OBFID = "CL_00001608";
 
-    public EntityAIWander(EntityCreature p_i1648_1_, double p_i1648_2_)
-    {
+    public EntityAIWander(EntityCreature p_i1648_1_, double p_i1648_2_) {
         this(p_i1648_1_, p_i1648_2_, 120);
     }
 
-    public EntityAIWander(EntityCreature p_i45887_1_, double p_i45887_2_, int p_i45887_4_)
-    {
+    public EntityAIWander(EntityCreature p_i45887_1_, double p_i45887_2_, int p_i45887_4_) {
         this.entity = p_i45887_1_;
         this.speed = p_i45887_2_;
         this.field_179481_f = p_i45887_4_;
         this.setMutexBits(1);
     }
 
-    public boolean shouldExecute()
-    {
-        if (!this.field_179482_g)
-        {
-            if (this.entity.getAge() >= 100)
-            {
+    public boolean shouldExecute() {
+        if (!this.field_179482_g) {
+            if (this.entity.getAge() >= 100) {
                 return false;
             }
 
-            if (this.entity.getRNG().nextInt(this.field_179481_f) != 0)
-            {
+            if (this.entity.getRNG().nextInt(this.field_179481_f) != 0) {
                 return false;
             }
         }
 
         Vec3 vec3 = RandomPositionGenerator.findRandomTarget(this.entity, 10, 7);
 
-        if (vec3 == null)
-        {
+        if (vec3 == null) {
             return false;
-        }
-        else
-        {
+        } else {
             this.xPosition = vec3.xCoord;
             this.yPosition = vec3.yCoord;
             this.zPosition = vec3.zCoord;
@@ -58,24 +58,19 @@
         }
     }
 
-    public boolean continueExecuting()
-    {
+    public boolean continueExecuting() {
         return !this.entity.getNavigator().noPath();
     }
 
-    public void startExecuting()
-    {
+    public void startExecuting() {
         this.entity.getNavigator().tryMoveToXYZ(this.xPosition, this.yPosition, this.zPosition, this.speed);
     }
 
-    public void func_179480_f()
-    {
+    public void func_179480_f() {
         this.field_179482_g = true;
     }
 
-    public void func_179479_b(int p_179479_1_)
-    {
+    public void func_179479_b(int p_179479_1_) {
         this.field_179481_f = p_179479_1_;
     }
-}
+}
